<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/img/4.jpg</normaloff>:/img/4.jpg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>挂号系统</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>451</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入患者姓名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="patient_name">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_14" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请选择挂号科室编号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="patient_department">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_15" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请选择挂号医生编号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="patient_doctor">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="register_btn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>400</y>
         <width>251</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认挂号</string>
       </property>
      </widget>
      <widget class="QLabel" name="picture_view">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>130</y>
         <width>411</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>排队情况</string>
      </attribute>
      <widget class="QTableWidget" name="queue_table">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>701</width>
         <height>751</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <row>
        <property name="text">
         <string>D1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D10</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D14</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>D15</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>B超室</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Doctor1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Doctor2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Doctor3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>科室排队人数</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </item>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>180</y>
         <width>391</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>10</y>
         <width>391</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLCDNumber" name="cur_time"/>
        </item>
        <item>
         <widget class="QPushButton" name="next_time_btn">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模拟下一分钟</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>记录查询</string>
      </attribute>
      <widget class="QTextBrowser" name="search_result">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>70</y>
         <width>501</width>
         <height>501</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>461</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_11" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入医生科室：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="doctor_department">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_16" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入医生编号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="doctor_num">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="search_btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>490</y>
         <width>251</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认查询</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
